tools/ci_build/github/azure-pipelines/templates/android-java-api-aar-test.yml

```diff
@@ -84,7 +84,6 @@ jobs:
     displayName: Run E2E tests using Browserstack
     workingDirectory: $(Build.BinariesDirectory)/android_test/android/app/build/outputs/apk
     timeoutInMinutes: 15
-    enabled: false
```

Comment:
please also update this comment:
https://github.com/microsoft/onnxruntime/blob/9ebfd7a9175e7ad2da78bfec9500b609397da845/tools/ci_build/github/azure-pipelines/templates/android-java-api-aar-test.yml#L75